docs/users/user_train.rst

```diff
@@ -0,0 +1,23 @@
+How to run
+==========
+
+This package provides users with the capabilities of running and simulating PPFL on either a single machine, a cluster, or heterogeneous machines.
+
+Serial run
+----------
+
+.. code-block:: console
+
+    $ python ./my_appfl.py
+
+
+MPI
+---
+
+.. code-block:: console
+
+    $ mpiexec -np 5 python ./my_appfl.py
+
+
+gRPC
+----
```

Comment:
@youngdae We need some detailed descriptions for users how to run/simulate `appfl` on multiple heterogeneous machines.